src/SFA.DAS.EmployerIncentives.Domain/Services/NewApprenticeIncentiveEligibilityService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Domain.Services;
+using SFA.DAS.EmployerIncentives.ValueObjects;
+
+namespace SFA.DAS.EmployerIncentives.Domain.Services
+{
+    public class NewApprenticeIncentiveEligibilityService : INewApprenticeIncentiveEligibilityService
+    {
+        public bool IsApprenticeshipEligible(Apprenticeship apprenticeship)
+        {
+            return NewApprenticeIncentive.NewApprenticeIncentive.IsApprenticeshipEligible(apprenticeship);
```

Comment:
Can we get read of the double NewApprenticeIncentive?  Maybe the namespace could be ApprenticeIncentive